packages/hooks/src/hooks/useWallet.ts

```diff
@@ -50,19 +50,7 @@ export function useWallet() {
         // If it is not, then install it into the user's MetaMask
         // @ts-expect-error unknown error type
         if (error.code === 4902) {
-          try {
-            await window.ethereum.request({
-              method: 'wallet_addEthereumChain',
-              params: [
-                {
-                  chainId: '0x61',
-                  rpcUrl: 'https://data-seed-prebsc-1-s1.binance.org:8545/',
-                },
-              ],
-            });
-          } catch (addError) {
-            console.error(addError);
-          }
+          alert('Please add the network to your MetaMask');
```

Comment:
`alert` is jarring UX, if we handle this shouldn't we handle it properly? How about setting a state variable and exposing it?

Reply:
Yeah alert kinda annoy users, so I think returning the error to the user or logging it to the console would work. WDYT @Dhaiwat10 ?

Reply:
The problem with only logging it is that non-technical users won't even know that there's an error. Alert is ugly but atleast it lets them know about what went wrong.